src/components/pages/search.js

```diff
@@ -35,14 +37,20 @@ function Search({ initialState }) {
 
   const [filteredDocs, setFilteredDocs] = useState([...sortedDocs]);
 
-  const [selectedTags, setSelectedTags] = useState([]);
+  const sanitizedQueryTags = new Set(decodedQuery.split(/\s/)
+    .filter(tag => tag.length > 2)
+    .map(tag => tag.toLowerCase()));
+
+  const allTags = docs.map(doc => doc.tags).flat();
+
+  const allUniqueTags = [...new Set(allTags)];
+
+  const initialSelectedTags = [...allUniqueTags]
```

Comment:
You don't need to spread here and make a new array, `allUniqueTags` is already an array now

Reply:
True, thanks.